plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2740,7 +2740,8 @@ function wc_update_910_add_launch_your_store_tour_option() {
  * Add woocommerce_hooked_blocks_version option for existing stores that are using a theme that supports the Block Hooks API
  */
 function wc_update_920_add_wc_hooked_blocks_version_option() {
-	if ( ! wc_current_theme_is_fse_theme() && ! current_theme_supports( 'block-template-parts' ) ) {
+	if ( ! wp_is_block_theme()
+ && ! current_theme_supports( 'block-template-parts' ) ) {
```

Comment:
I won't list every time the formatting is bad, but it looks like you need to fix code formatting across the PHP files :)